include/unifex/finally.hpp

```diff
@@ -590,16 +590,17 @@ namespace unifex
         manual_lifetime_union<std::exception_ptr, Errors...>;
 
     public:
-      template <typename CompletionSender2, typename Receiver2>
+      type() = default;
```

Comment:
Why was it necessary to declare a default constructor here?
I wouldn't expect it to ever be called.

Reply:
at one point the variadic constructor was eating the default constructor call as well. I am using the std::in_place suggestion now.